src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,17 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.service.suppliers.FigureSupplier;
+
 public class HelloWorld {
+    private static FigureSupplier figureSupplier = new FigureSupplier();
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+            figures[i].draw();
+        }
```

Comment:
half of figures must be defaults
<img width="808" alt="Screen Shot 2022-10-27 at 9 45 05 PM" src="https://user-images.githubusercontent.com/90692308/198373157-efba44f3-e1ab-4d2e-8caf-2b0b99a3f0f2.png">